test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1530,6 +1536,10 @@ func scheduleGpuPod(f *framework.Framework, id string) error {
 		Labels:         map[string]string{"requires-gpu": "yes"},
 	}
 
+	if gpuType != "" {
+		config.NodeSelector = map[string]string{"cloud.google.com/gke-accelerator": gpuType}
```

Comment:
Sorry for late comment, but can you make the label name const?

Reply:
Done